harpy/simulate_linkedreads.py

```
@@ -0,0 +1,112 @@
+
+    check_fasta(genome_hap1)
+    check_fasta(genome_hap2)
+    if barcodes:
+        bc_len = validate_barcodefile(barcodes, True, quiet)
+    os.makedirs(f"{workflowdir}/", exist_ok= True)
+    fetch_rule(workflowdir, "simulate_linkedreads.smk")
+    fetch_script(workflowdir, "HaploSim.pl")
+    os.makedirs(f"{output_dir}/logs/snakemake", exist_ok = True)
+    sm_log = snakemake_log(output_dir, "simulate_linkedreads")
+    conda_envs = ["simulations"]
+    configs = {
```

Ruff (0.8.2)

Local variable `configs` is assigned to but never used

Remove assignment to unused variable `configs`

(F841)